sunpy/map/mapbase.py

```diff
@@ -2708,6 +2709,7 @@ def plot(self, *, annotate=True, axes=None, title=True, autoalign=False,
 
         return ret
 
+    @deprecated(since="6.1", message="The 'contour' method is deprecated and will be removed in a future release. Please use 'get_contours'.", alternative="sunpy.map.GenericMap.get_contours")
     def contour(self, level, **kwargs):
         """
         Returns coordinates of the contours for a given level value.
```

Comment:
I can't leave a comment at the right location now but since we added the deprecated, the example now fails the CI tests. 

I think we can add a IGNORE_WANRINGS flag to doctest to ignore the warning. Hopefully that will work.